doc/Advanced-configuration.md

```diff
@@ -319,6 +319,31 @@ An example configuration can look as follows:
 `{odbc_server, {pgsql, "localhost", "username", "database", "pass",
                [{ssl, required}, {ssl_opts, [{verify, verify_peer}, {cacertfile, "path/to/cacert.pem"}]}]}}.`
 
+### ODBC SSL connection setup
+
+If you've configured MongooseIM to use an ODBC driver, i.e. you've provided an ODBC connection string to `odbc_server` option, e.g.
```

Comment:
`odbc` in `odbc_server` really needs to go, it's very confusing that normally you would *not* use ODBC driver but still configure things with `odbc_*` options.

Reply:
Good point, that's could be the subject of another PR. @fenek @michalwski

Reply:
Definitely, it is something that has been annoying me for quite a long time.